[2022-04-13 22:53:29,250] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: LocalIngestionDAG.ingest scheduled__2022-02-02T06:00:59+00:00 [queued]>
[2022-04-13 22:53:29,262] {taskinstance.py:1032} INFO - Dependencies all met for <TaskInstance: LocalIngestionDAG.ingest scheduled__2022-02-02T06:00:59+00:00 [queued]>
[2022-04-13 22:53:29,262] {taskinstance.py:1238} INFO - 
--------------------------------------------------------------------------------
[2022-04-13 22:53:29,263] {taskinstance.py:1239} INFO - Starting attempt 2 of 2
[2022-04-13 22:53:29,263] {taskinstance.py:1240} INFO - 
--------------------------------------------------------------------------------
[2022-04-13 22:53:29,277] {taskinstance.py:1259} INFO - Executing <Task(BashOperator): ingest> on 2022-02-02 06:00:59+00:00
[2022-04-13 22:53:29,281] {standard_task_runner.py:52} INFO - Started process 12595 to run task
[2022-04-13 22:53:29,284] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'LocalIngestionDAG', 'ingest', 'scheduled__2022-02-02T06:00:59+00:00', '--job-id', '2020', '--raw', '--subdir', 'DAGS_FOLDER/data_ingestion_local.py', '--cfg-path', '/tmp/tmpxok1znvw', '--error-file', '/tmp/tmpqlmvnjho']
[2022-04-13 22:53:29,285] {standard_task_runner.py:77} INFO - Job 2020: Subtask ingest
[2022-04-13 22:53:29,341] {logging_mixin.py:109} INFO - Running <TaskInstance: LocalIngestionDAG.ingest scheduled__2022-02-02T06:00:59+00:00 [running]> on host 13db72185839
[2022-04-13 22:53:29,402] {taskinstance.py:1426} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=LocalIngestionDAG
AIRFLOW_CTX_TASK_ID=ingest
AIRFLOW_CTX_EXECUTION_DATE=2022-02-02T06:00:59+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-02-02T06:00:59+00:00
[2022-04-13 22:53:29,404] {subprocess.py:62} INFO - Tmp dir root location: 
 /tmp
[2022-04-13 22:53:29,405] {subprocess.py:74} INFO - Running command: ['bash', '-c', 'ls /opt/***']
[2022-04-13 22:53:29,415] {subprocess.py:85} INFO - Output:
[2022-04-13 22:53:29,419] {subprocess.py:89} INFO - ***.cfg
[2022-04-13 22:53:29,420] {subprocess.py:89} INFO - dags
[2022-04-13 22:53:29,421] {subprocess.py:89} INFO - logs
[2022-04-13 22:53:29,421] {subprocess.py:89} INFO - output_2021-02.csv
[2022-04-13 22:53:29,422] {subprocess.py:89} INFO - output_2022-02.csv
[2022-04-13 22:53:29,423] {subprocess.py:89} INFO - output_2022-04.csv
[2022-04-13 22:53:29,423] {subprocess.py:89} INFO - plugins
[2022-04-13 22:53:29,424] {subprocess.py:89} INFO - requirements.txt
[2022-04-13 22:53:29,425] {subprocess.py:89} INFO - scripts
[2022-04-13 22:53:29,425] {subprocess.py:89} INFO - webserver_config.py
[2022-04-13 22:53:29,426] {subprocess.py:93} INFO - Command exited with return code 0
[2022-04-13 22:53:29,458] {taskinstance.py:1277} INFO - Marking task as SUCCESS. dag_id=LocalIngestionDAG, task_id=ingest, execution_date=20220202T060059, start_date=20220413T225329, end_date=20220413T225329
[2022-04-13 22:53:29,496] {local_task_job.py:154} INFO - Task exited with return code 0
[2022-04-13 22:53:29,533] {local_task_job.py:264} INFO - 0 downstream tasks scheduled from follow-on schedule check
